docs/landing-zone-v2/README.md

```diff
@@ -78,13 +69,13 @@ To deploy this Landing Zone you will need to:
 
   1. [Complete the bootstrap procedure.](#1-complete-the-bootstrap-procedure)
   2. [Create your landing zone.](#2-create-your-landing-zone)
-  3. [Deploy the infrastructure using either kpt or gitops-git or gitops-oci.](#3-deploy-the-infrastructure-using-either-kpt-or-gitops-git-or-gitops-oci)
+  3. [Deploy the infrastructure using either gitops](#deploy-the-infrastructure-using-gitops)
```

Comment:
the word "either" is not required

Reply:
Will get that fixed this morning. Thanks for the review Dave!